2019-02-21 05:07:00,333 INFO  execution-log - 	Initializing control map with map # MASTER
----------------
2019-02-21 05:07:00,339 INFO  execution-log - Initializing Test Automation
----------------
2019-02-21 05:07:00,343 INFO  execution-log - 	target browser: Chrome
----------------
2019-02-21 05:07:00,347 INFO  execution-log - TC # : 4, Downloading test case from DB
----------------
2019-02-21 05:07:00,356 INFO  execution-log - TC # : 4, Getting test data from DB
----------------
2019-02-21 05:07:00,356 INFO  execution-log - TC # : 4, Getting control map :MASTER from DB
----------------
2019-02-21 05:07:00,366 INFO  execution-log - TC # : 4, Test data retrieval completed
----------------
2019-02-21 05:07:00,370 INFO  execution-log - TC # : 4, Control map retrieval completed
----------------
2019-02-21 05:07:00,375 INFO  execution-log - TC # : 4, Parsing commands
----------------
2019-02-21 05:07:00,381 INFO  execution-log - Loading captured values from persistence.
----------------
2019-02-21 05:07:00,385 INFO  execution-log - TC # : 4, Started executing test case in the VMTESTPronto environment (https://www.prontoinsurance.com/)
----------------
2019-02-21 05:07:00,401 INFO  execution-log -  Beginning StartIterationCommand ( Iterations : 1 )
----------------
2019-02-21 05:07:00,406 INFO  execution-log - TC # : 4, Starting iteration #1
----------------
2019-02-21 05:07:00,410 INFO  execution-log - 	 Command completed.
----------------
2019-02-21 05:07:00,414 INFO  execution-log -  Beginning InvokeApplicationCommand ( Target :  )
----------------
2019-02-21 05:07:01,886 INFO  execution-log - 	 Command completed.
----------------
2019-02-21 05:07:01,898 INFO  execution-log -  Beginning PageCommand ( CurrentPage : Pronto Home, FlowIdentifier : 1, DiIndicator : DI )
----------------
2019-02-21 05:07:01,903 INFO  execution-log - 	 Command completed.
----------------
2019-02-21 05:07:01,907 INFO  execution-log -  Beginning LoadTestDataItemCommand ( CurrentPage : Pronto Home, FlowIdentifier : 1, DiIndicator : DI )
----------------
2019-02-21 05:07:01,920 INFO  execution-log - [Sequence No - 1] Beginning ValidateExistStatusCommand ( LogicalFieldName : GET A QUOTE, LogicalFieldValue : True )
----------------
2019-02-21 05:07:01,939 INFO  execution-log - [Sequence No - 1]	 Command completed.
----------------
2019-02-21 05:07:01,950 INFO  execution-log - [Sequence No - 2] Beginning ActionCommand ( LogicalFieldName : GET A QUOTE, LogicalFieldValue : Click )
----------------
2019-02-21 05:07:02,062 INFO  execution-log - [Sequence No - 2]	 Command completed.
----------------
2019-02-21 05:07:02,072 INFO  execution-log - [Sequence No - 3] Beginning ValidateExistStatusCommand ( LogicalFieldName : CALIFORNIA, LogicalFieldValue : True )
----------------
2019-02-21 05:07:02,095 INFO  execution-log - [Sequence No - 3]	 Command completed.
----------------
2019-02-21 05:07:02,107 INFO  execution-log - [Sequence No - 4] Beginning ActionCommand ( LogicalFieldName : CALIFORNIA, LogicalFieldValue : Click )
----------------
2019-02-21 05:07:15,319 ERROR execution-log - TC # : 4, Timed out after 10 seconds
----------------
2019-02-21 05:07:23,341 ERROR execution-log - TC # : 4, Timed out after 10 seconds
----------------
2019-02-21 05:07:23,347 ERROR execution-log - TC # : 4, No more iterations to run. Test execution will abort.
----------------
2019-02-21 05:07:23,353 ERROR execution-log - TC # : 4, Framework failed to run the test due to a test configuration error.
	VM.Platform.TestAutomationFramework.Core.Exceptions.WorkflowFailedException: Timed out after 10 seconds ---> OpenQA.Selenium.WebDriverTimeoutException: Timed out after 10 seconds
   at OpenQA.Selenium.Support.UI.DefaultWait`1.ThrowTimeoutException(String exceptionMessage, Exception lastException)
   at OpenQA.Selenium.Support.UI.DefaultWait`1.Until[TResult](Func`2 condition)
   at VM.Platform.TestAutomationFramework.Adapters.Selenium.SeleniumAdapter.WaitForElementToBeClickable(ControlDefinition ctrlDefinition) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Adapters.Selenium\SeleniumAdapter.cs:line 713
   at VM.Platform.TestAutomationFramework.UIAutomation.Commands.Interactions.ActionCommand.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.UIAutomation\Commands\Interactions\ActionCommand.cs:line 30
   --- End of inner exception stack trace ---
   at VM.Platform.TestAutomationFramework.UIAutomation.Commands.Interactions.ActionCommand.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.UIAutomation\Commands\Interactions\ActionCommand.cs:line 42
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandDecorator.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandDecorator.cs:line 22
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandLogger.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandLogger.cs:line 93
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandDecorator.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandDecorator.cs:line 22
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandErrorHandler.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandErrorHandler.cs:line 65
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandDecorator.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandDecorator.cs:line 22
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandTimer.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandTimer.cs:line 26
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandDecorator.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandDecorator.cs:line 22
   at VM.Platform.TestAutomationFramework.Core.Decorators.IgnorePagesDecorator.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\IgnorePagesDecorator.cs:line 24
   at VM.Platform.TestAutomationFramework.UIAutomation.Commands.Workflow.LoadTestDataItemCommand.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.UIAutomation\Commands\Workflow\LoadTestDataItemCommand.cs:line 96
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandDecorator.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandDecorator.cs:line 22
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandLogger.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandLogger.cs:line 93
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandDecorator.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandDecorator.cs:line 22
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandErrorHandler.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandErrorHandler.cs:line 65
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandDecorator.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandDecorator.cs:line 22
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandTimer.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandTimer.cs:line 26
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandDecorator.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandDecorator.cs:line 22
   at VM.Platform.TestAutomationFramework.Core.Decorators.IgnorePagesDecorator.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\IgnorePagesDecorator.cs:line 24
   at VM.Platform.TestAutomationFramework.Core.TestCommandRunner.Execute(TestRunContext testRunContext) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\TestCommandRunner.cs:line 75
   at VM.Platform.TestAutomationFramework.Core.SingleTestCaseRunner.ExecuteSingleTestCase(Int32 testCaseId, String targetEnvironment, String repositiory, Int32 Retry) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\SingleTestCaseRunner.cs:line 167
----------------
2019-02-21 05:07:23,371 INFO  execution-log - Saving captured values to file.
----------------
2019-02-21 05:07:23,379 ERROR execution-log - TC # : 4,  has failed
----------------
